workspaces/vscode-python/Dockerfile

```diff
@@ -66,39 +28,15 @@ RUN export arch=`uname -m` \
  && mamba clean --all --yes --quiet \
  && pip cache purge
 
-# Install VS Code extensions and clear the extension cache to reduce image size.
+# Install VS Code extensions.
 # We have first installed Python in the previous step, as specified in the
 # documentation of the ms-python.python extension.
 USER coder
 RUN code-server --disable-telemetry --force \
     # vscode support for python, including debugger
     --install-extension ms-python.python \
     # auto-fix indentation for multiline contexts
-    --install-extension KevinRose.vsc-python-indent \
-    # visualize csv files in vscode window
-    --install-extension mechatroner.rainbow-csv \
-    # format tables in markdown files
-    --install-extension fcrespo82.markdown-table-formatter \
-    && rm -rf /home/coder/.local/share/code-server/CachedExtensionVSIXs
-
-# We standardize on /home/coder/workspace as the default working directory
-# for the editor, and we recommend that staff set this as workspaceOptions
-# "home" in the question configuration. See the README.md for details.
-USER coder
-RUN mkdir -p "/home/coder/workspace" "/home/coder/.local/share/code-server/User"
-WORKDIR "/home/coder/workspace"
-COPY --chmod=0644 --chown=coder:coder settings.json tasks.json /home/coder/.local/share/code-server/User/
-
-# Prepare the entrypoint helper that steps down to a limited user in local dev mode
-COPY --chmod=0755 --chown=root:root pl-gosu-helper.sh /usr/bin/
+    --install-extension KevinRose.vsc-python-indent
 
-# Make sure that code-server's entrypoint doesn't run fixuid unnecessarily
-USER root
-RUN mkdir -p /run /var/run && \
-    touch /run/fixuid.ran /var/run/fixuid.ran
-
-USER coder
-EXPOSE 8080
-ENTRYPOINT ["/usr/bin/pl-gosu-helper.sh", "/usr/bin/entrypoint.sh", "--auth", "none", \
-    "--disable-update-check", "--disable-telemetry", \
-    "--bind-addr", "0.0.0.0:8080", "."]
+# USAGE:
+# docker build --platform linux/amd64 -t prairielearn/workspace-vscode-python .
```

Comment:
If you want this anywhere, please put it in the readme, though I don't think this is necessary.

Reply:
Usage removed from Dockerfiles.